grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovementListItem.groovy

```diff
@@ -103,12 +104,14 @@ class OutboundStockMovementListItem implements Serializable {
     }
 
     Map toJson() {
+        def g = ApplicationHolder.application.mainContext.getBean('org.codehaus.groovy.grails.plugins.web.taglib.ApplicationTagLib')
         return [
                 id                  : id,
                 name                : name,
                 description         : description,
                 statusCode          : statusCode?.toString(),
                 statusVariant       : status?.variant?.name,
+                statusLabel         : "${g.message(code: 'enum.RequisitionStatus.' + status.name())}",
```

Comment:
Since `status` on `OutboundStockMovementListItem` is a **RequisitionStatus**, I feel like we can use the `mapStatusCodes()` _or whatever the name for it will be_, to extract the label from it.
I am not sure if it is going to work but lets try doing something like

```suggestion
                statusLabel         : status.mapStatusCodes()?.label,
```

Reply:
Yup, it works